From 0941a15e9e9eab3a3756d1ceb8c347a351b23f54 Mon Sep 17 00:00:00 2001
From: Darrell Kavanagh <darrell.kavanagh@gmail.com>
Date: Sun, 5 Feb 2023 20:57:12 +0000
Subject: [PATCH] Add hwdb sensor entry for Lenovo IdeaPad Duet 3 10IGL5
 (82AT).

(cherry picked from commit 6823b5bb99b605090d967d08586a4ce8fd5f989f)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 31f7bf031d..044fecdb38 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -648,6 +648,10 @@ sensor:modalias:i2c:bmc150_accel:dmi:*:svnLENOVO:*:pvrLenovoYoga300-11IBR:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, 1, 0; 0, 0, -1
  ACCEL_LOCATION=base
 
+# IdeaPad Duet 3 10IGL5 (82AT)
+sensor:modalias:acpi:SMO8B30*:dmi:*:svnLENOVO*:pn82AT:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
+
 #########################################
 # LINX
 #########################################
